src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1117,4 +1116,150 @@ private class ACCT_IndividualAccounts_TEST {
         List<Contact> returnedContactDelete = [SELECT Id, IsDeleted FROM Contact WHERE Id = :con.Id];
         System.assertEquals(0, returnedContactDelete.size());
     }
+
+    /*********************************************************************************************************
+    * @description Test Method to test sorting Contacts alphabetically by LastName FirstName
+    * with {!{!FirstName}} {!LastName} Household format.
+    */
+    @isTest
+    private static void sortContactsFirstLastNames() {
```

Comment:
These test methods are great. What happens if one of the First Names is null/blank? (I think only Last Name is required...)

Reply:
Added new test method sortContactsWithoutFirstNames() to test blank firstnames.